config/pos.php

```diff
@@ -100,7 +95,7 @@
         ],
         'yapikredi' => [
             'name'  => 'Yapıkredi',
-            'class' => Mews\Pos\PosNet::class,
+            'class' => Mews\Pos\Gateways\PosNet::class,
             'urls'  => [
                 'production'    => 'https://www.posnet.ykb.com/PosnetWebService/XML',
```

Comment:
Yapı Kredi için yeni adresler;

`Production : https://posnet.yapikredi.com.tr/PosnetWebService/XML`
`Test : https://setmpos.yapikredi.com.tr/PosnetWebService/XML`
`Gateway Production : https://www.posnet.yapikredi.com.tr/3DSWebService/YKBPaymentService`
`Gateway Test : https://setmpos.yapikredi.com.tr/3DSWebService/YKBPaymentService`

Reply:
İlgili adres için Yapı Kredi mail içeriği;

Merhaba,

Posnet.ykb.com adresinin sertifikasının geçerlilik tarihi sone ermiş olup, bu adres yerine aşağıdaki adreslerimize artık işlem gönderecek şekilde kendi tarafınızda güncelleme yapmanız gerekmektedir.

TDS Web Servisi (OOS_TDS_SERVICE_URL) ==> https://posnet.yapikredi.com.tr/3DSWebService/YKBPaymentService
XML Web Servisi (XML_SERVICE_URL) ==> https://posnet.yapikredi.com.tr/PosnetWebService/XML

Mevcutta işlemlerinizi gönderdiğiniz Posnet adreslerinizi posnet.yapikredi.com.tr olarak değiştirmeniz ve sertifikaları yüklemeniz gerekmektedir. Gerekli sertifikalara aşağıdaki adresten ulaşabilirsiniz.

https://ykb.sharefile.com/d-sd7c36455a2540cda

İyi çalışmalar,

Reply:
surekli degisiyor bu Yapikredi gateway'i. Daha nissan ayinda guncellemistim. Oneriniz icin tesekkurler, guncelledim.